carmm/build/mixed_oxide.py

```diff
@@ -0,0 +1,96 @@
+
+def mixed_oxide(atoms_object:Atoms, metal1, metal2, ratio, metal1_charge, metal2_charge, metal1_moment, metal2_moment, up_or_down_spin=None):
+    '''
+    Provided an Atoms object of a metal oxide with formula AxOy, this function creates another
+    Atoms object for a mixed oxide with formula Ax-kBkOy where A and B are the two metals.
+    x is the initial stoichiometry of A in AxOy.
+    The value of k can be figured out from the equation ratio(metal_A:metal_B) = x-k/k for a given value of ratio
+    IMPORTANT NOTE: As of now this functionality only works for oxides which the metal A is in only one oxidation state
+    like CoO or Mn2O3, etc.
+
+    :param geometry_file: The geometry.in file for the oxide AxOy with initial moments and charges specified.
```

Comment:
All these inputs need to state there variable type - currently it is unclear for a new user. See other functions for how this should be defined usefully.